packages/terra-clinical-header/src/Header.module.scss

```diff
@@ -1,77 +1,80 @@
-    padding: var(--terra-clinical-header-padding, 0.714rem);
+    min-height: var(--terra-clinical-header-min-height, 2.92857rem);
+    padding: var(--terra-clinical-header-padding, 0.35714rem 0.35714rem 0.35714rem 0);
     position: relative;
     width: 100%;
 
     .flex-collapse {
-      align-items: center;
+      align-self: var(--terra-clinical-header-flex-collapse-align-self);
       display: flex;
       flex: 0 0 auto;
       height: 100%;
+      margin: var(--terra-clinical-header-flex-collapse-margin);
       max-width: 40%;
       position: relative;
     }
   }
 
+  .subheader {
+    background-color: var(--terra-clinical-header-subheader-background-color, #c8cacb);
+    // color: var(--terra-clinical-header-subheader-color);
+    min-height: var(--terra-clinical-header-subheader-min-height, 2.857rem);
+    padding: var(--terra-clinical-header-subheader-padding, 0.3571rem 0.7143rem);
+  }
+
   .flex-fill {
     align-self: center;
     flex: 1 1 auto;
-    height: 100%;
-    min-width: 0;
     position: relative;
     width: 100%;
   }
 
   .flex-end {
-    align-items: center;
+    align-self: var(--terra-clinical-header-flex-end-align-self);
     display: flex;
     flex: 0 0 auto;
-    height: 100%;
     position: relative;
   }
 
   .title-container {
-    height: 100%;
     position: relative;
     width: 100%;
   }
 
   .title {
-    font-size: var(--terra-clinical-header-title-font-size, 1.285rem);
-    font-weight: normal;
-    height: 100%;
-    line-height: var(--terra-clinical-header-title-line-height, 1.75);
+    font-size: var(--terra-clinical-header-title-font-size, 1.10714rem);
+    font-weight: var(--terra-clinical-header-title-font-weight, 500);
+    hyphens: auto;
     margin-bottom: 0;
     margin-top: 0;
-    overflow: hidden;
-    padding: var(--terra-clinical-header-title-padding, 0);
-    text-overflow: ellipsis;
-    white-space: nowrap;
+    overflow-wrap: break-word; /* Modern browsers */
+    padding: var(--terra-clinical-header-title-padding, 0.28571rem 0.71428rem 0 0.71428rem);
     width: 100%;
-    word-break: normal;
-    word-wrap: normal;
+    word-wrap: break-word; /* For IE 10 and IE 11 */
   }
-
-  .flex-subheader {
-    background-color: var(--terra-clinical-header-subheader-background-color, #c8cacb);
-    border-bottom-color: currentColor;
-    border-bottom-style: none;
-    border-bottom-width: 0;
-    display: flex;
-    height: var(--terra-clinical-header-subheader-height, 2.857rem);
-    justify-content: space-between;
-    overflow: hidden;
-    padding: 0.3571rem 0.7143rem;
-    position: relative;
-    width: 100%;
+  
+  /* stylelint-disable selector-max-compound-selectors */
+  .flex-end + .flex-fill {
+    .title {
+      padding-left: var(--terra-clinical-header-end-content-plus-title-padding-left, 0.35714rem);
+    }
+  }
+  
+  .flex-fill + .flex-end {
+    .title {
+      padding-right: var(--terra-clinical-header-title-plus-end-content-padding-right, 0.35714rem);
+    }
```

Comment:
@jeremyfuksa can you config these changes are desired to match: https://github.com/cerner/terra-core/pull/2714/files?file-filters%5B%5D=.md&file-filters%5B%5D=.png&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-bf4aa35fd6c961609389f4c3de68798bR62-R67

I assume we wanted to pad both sides since we don't have knowledge on what content is being placed in the start/end content boxes. This isn't needed for terra-action-header since it controls the padded styles on the button content it drops in here: https://github.com/cerner/terra-core/blob/master/packages/terra-action-header/src/ActionHeader.module.scss#L4

Reply:
I think you're right in padding both sides.